rss-reader/Model/Entity/FeedItem/FeedItem.m

```diff
@@ -0,0 +1,80 @@
+//
+//  FeedItem.m
+//  rss-reader
+//
+//  Created by Uladzislau on 11/17/20.
+//
+
+#import "FeedItem.h"
+#import "NSDate+StringConvertible.h"
+
+NSString *const kRSSItem = @"item";
+NSString *const kRSSItemTitle = @"title";
+NSString *const kRSSItemLink = @"link";
+NSString *const kRSSItemSummary = @"description";
+NSString *const kRSSItemCategory = @"category";
+NSString *const kRSSItemPubDate = @"pubDate";
+
+@interface FeedItem ()
+
+@property (nonatomic, copy, readwrite) NSString *title;
+@property (nonatomic, copy, readwrite) NSString *link;
+@property (nonatomic, copy, readwrite) NSString *summary;
+@property (nonatomic, copy, readwrite) NSString *category;
+@property (nonatomic, retain, readwrite) NSDate *pubDate;
+@property (nonatomic, retain, readwrite) NSArray<MediaContent *> *mediaContent;
+
+@end
+
+@implementation FeedItem
+
++ (instancetype)objectWithDictionary:(NSDictionary *)dictionary {
```

Comment:
? Probably more clearly if name it `itemWithDictionary`

Reply:
no